blog/2021-06-07-release-notes.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Luigi v1.13.0
+seoMetaDescription: Release notes for Luigi v1.13.0
+author:
+  - Aleksandra Simeonova
+layout: blog
+---
+
+Luigi 1.13.0 is the latest Luigi release. It includes updates of Fundamental Library Styles, new options in the user settings, as well as other features and bug fixes. Read the release notes below to learn more.
+
+<!-- Excerpt -->
+
+#### Breaking changes in Fundamental Library Styles
+
+With Luigi v1.13.0, the new v0.17.0 of Fundamental Library Styles was included. It led to having breaking changes in the Luigi Dialog/Modal layout and Shellbar Counter. New HTML attributes were added to those components to increase accessibility. You can see the new layout of these components here: [Dialog](https://sap.github.io/fundamental-styles/?path=/docs/components-dialog--default-dialog) and [Shellbar Counter](https://sap.github.io/fundamental-styles/?path=/docs/components-shellbar--primary).
```

Comment:
Maybe we could add sth. like
"As always this does only affect you if you have customized fundamentals css in your app" or sth. like that